app/forms/field_handlers/mobile_number_handler.py

```diff
@@ -10,7 +10,7 @@ class MobileNumberHandler(FieldHandler):
     MANDATORY_MESSAGE_KEY = "MANDATORY_MOBILE_NUMBER"
 
     @cached_property
-    def validators(self):
+    def validators(self) -> list:
```

Comment:
I think it may be possible to be more specific with the occurrences of `list` types in this PR, for example `List[Union[ResponseRequired, MobileNumberCheck]]` here.

Reply:
Added more specific type hints for returned lists of validators, thanks!